src/+otp/+pendulum/PendulumProblem.m

```diff
@@ -36,16 +36,18 @@ function onSettingsChanged(obj)
         function validateNewState(obj, newTimeSpan, newY0, newParameters)
             validateNewState@otp.Problem(obj, newTimeSpan, newY0, newParameters)
             
-            %            otp.utils.StructParser(newParameters) ...
-            %                 .checkField('g', 'scalar', 'real', 'finite', 'positive') ...
-            %                 .checkField('lengths', 'vector', 'real', 'finite', 'positive');
+            otp.utils.StructParser(newParameters) ...
+                .checkField('g', 'scalar', 'real', 'finite', 'positive') ...
+                .checkField('lengths', 'vector', 'real', 'finite', 'positive') ...
+                .checkField('masses',  'vector', 'real', 'finite', 'positive');
         end
         
-        function label = internalIndex2label(~, index)
-            if index == 1
-                label = 'Theta';
-            else
-                label = 'Omega';
+        function label = internalIndex2label(obj, index)
+            numPendulums = obj.NumVars/2;
+            if index <= numPendulums
+                label = sprintf('\\theta_{%d}', numPendulums);
+            else 
+                label = sprintf('\\omega_{%d}', index - numPendulums);
```

Comment:
I like these names, but would 'Angle' and 'Angular velocity' be more descriptive? I'm fine with keeping this.

Reply:
The latex displays nicely on plots.  `label2index` is not really used outside of plotting.